app/styles/_components.scss

```diff
@@ -528,6 +528,10 @@
     background-clip: padding-box;
 }
 
+.text-left {
+    text-align: left;
+}
```

Comment:
This is not my _favorite_ because I'd rather get rid of all of this global css, so this just feels like tech debt to me.